Makefile

```diff
@@ -76,4 +76,7 @@ issues: ## Run GitHub issue analysis scripts
 	./hack/github/feature_request_reactions.py > "karpenter-feature-requests-$(shell date +"%Y-%m-%d").csv"
 	./hack/github/label_issue_count.py > "karpenter-labels-$(shell date +"%Y-%m-%d").csv"
 
-.PHONY: help dev ci release test battletest verify codegen apply delete toolchain release licenses issues
+docs: ## Serve the docs site locally
```

Comment:
WDYT about calling this `website` to be consistent with the directory name? Specially since the website is a bit more than just docs.

Reply:
+1

Reply:
good idea, done!